ci/pipelines/perf-tests.yml

```diff
@@ -158,12 +147,15 @@ groups:
 definitions:
   - &scale-down-services
     in_parallel:
-      - put: scale-down-stubs
-        resource: paas
+      - task: scale-down-stubs
+        file: pay-ci/ci/tasks/scale-fargate-service.yml
         params:
-          command: scale
-          app_name: pay-stubs
-          instances: ((.:service-volumes.scale_down_to.stubs))
+          SERVICE_NAME: stubs
+          SCALE_DIRECTION: in
+          DESIRED_HEALTHY_INSTANCES: ((.:service-volumes.scale_down_to.stubs))
+          AWS_ACCESS_KEY_ID: ((.:role.AWS_ACCESS_KEY_ID))
```

Comment:
These wont work because the roles assumed by these creds are in other AWS accounts (the roles assumed are in test, staging, or production), but stubs is in the deploy account

I'd be interested to see if this works without providing the AWS creds (meaning concourse already has the permission it needs). Otherwise you might need to assume a separate role for just this service

Reply:
Are you sure? All the other services are the same.

Reply:
Yeah, all the other services are in the test account, but this ECS service is in the deploy account. Feel free to apply it and test it out though :)

Reply:
Ah, yes!